CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.2.0
+
+* Per-platform binaries will be available as Github release assets.
+* Documentation examples can be verified via `make test-heavy`.
```

Comment:
Sorry for not saying this in an earlier comment. If we want to nitpick, the "Documentation examples" are really the "zq examples under `zql/docs`" and nothing more. The examples in the top-level README are not verified, for example. You don't have to change this if it's not important.